corehq/apps/accounting/management/commands/change_role_for_software_plan_version.py

```diff
@@ -0,0 +1,89 @@
+import logging
+
+from django.core.management import BaseCommand
+
+from django_prbac.models import Role
+
+from corehq.apps.accounting.models import SoftwarePlanVersion
+
+logger = logging.getLogger(__name__)
+
+
+class OldRoleDoesNotExist(Exception):
+    pass
+
+
+class NewRoleDoesNotExist(Exception):
+    pass
+
+
+def change_role_for_software_plan_version(old_role, new_role, limit_to_plans=None, dry_run=False):
+    """
+    We typically do not support modifying SoftwarePlanVersions directly, and instead encourage creating a new one.
+    This command should only be used when it seems appropriate. The most typical use case would be when it is
+    desirable to delete the Role(slug=old_role) object entirely, and using this command to ensure no software
+    plan versions reference that old role.
+    :param old_role: slug for role to search for
+    :param new_role: slug for role that new software plan version should reference
+    :param limit_to_plans: optionally limit to specific plan names
+    :param dry_run: if False, will make changes to the DB
```

Comment:
*if False will not make changes to the DB

Reply:
I think it is correct the way it is worded currently.